### PR TITLE
Fix generateTexture params order

### PR DESCRIPTION
**Pull request by tong**

### CODE REVIEW

File: pixi/core/textures/RenderTexture.hx [hunk 1]
```diff
@@ -49,8 +49,8 @@ extern class RenderTexture extends Texture {
 	 * @param [scaleMode] {number} See {@link SCALE_MODES} for possible values
 	 * @param [resolution=1] {number} The resolution of the texture being generated
 	 */
-	@:overload(function(renderer:CanvasRenderer, ?width:Float, ?height:Float, ?scaleMode:Int, ?resolution:Float):Void {})
-	function new(renderer:WebGLRenderer, ?width:Float, ?height:Float, ?scaleMode:Int, ?resolution:Float);
+	@:overload(function(?width:Float, ?height:Float, renderer:CanvasRenderer, ?scaleMode:Int, ?resolution:Float):Void {})
+	function new(?width:Float, ?height:Float, ?renderer:WebGLRenderer, ?scaleMode:Int, ?resolution:Float);
```
Comment (adireddy) [inline pixi/core/textures/RenderTexture.hx:53]: Hey @tong, any reason for changing the order and making `renderer` optional?

I just checked pixi class https://github.com/pixijs/pixi.js/blob/dev/src/core/textures/RenderTexture.js and `renderer` is not optional.

Comment (adireddy) [inline pixi/core/textures/RenderTexture.hx]: Hey @tong, any reason for changing the order and making `renderer` optional?

I just checked pixi class https://github.com/pixijs/pixi.js/blob/dev/src/core/textures/RenderTexture.js and `renderer` is not optional.

**Comment by tong** [comment]
Now i am confused and wondering which version i was looking at.

**Comment by tong** [comment]
It was:
http://www.goodboydigital.com/pixijs/docs/classes/DisplayObject.html#method_generateTexture
Seems the docs are out of date. Sorry + nevermind request.

**Comment by adireddy** [comment]
You are looking at v2 docs.

v3 docs are here http://pixijs.github.io/docs/

**Pull request closed without merging**